compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/FlingBehavior.kt

```diff
@@ -43,7 +43,13 @@ interface FlingBehavior {
      * @return remaining velocity after fling operation has ended
      */
     suspend fun ScrollScope.performFling(initialVelocity: Float): Float
+
+    /**
+     * Update the internal parameters of FlingBehavior in accordance with the new [androidx.compose.ui.unit.Density] value.
+     *
+     * @param density new density value.
+     */
+    fun updateDensity(density: Density) = Unit
```

Comment:
Instead of adding the method to the public interface, we can define an expect class:
```
internal expect class DefaultFlingBehavior : FlingBehavior {
    expect suspend fun ScrollScope.performFling(initialVelocity: Float): Float
    expect fun updateDensity(density: Density)
}
```
or just also as interface:
```
internal interface DefaultFlingBehavior : FlingBehavior {
    fun updateDensity(density: Density)
}
```
Also (just for information):
1. It is a public common API change, we shouldn't change it without a strong reason and changing it in AOSP first.
2. `FlingBehavior` can depend not only on Density but on other external things. Considering this, it is better for public API to not expose setters/updaters for all kinds of properties, and ask implementations to subscribe  via constructor's. For example:
```
CupertinoFlingBehaviour(
    currentDensity = { currentLocalOf(LocalDensity) }
)
```
or
```
CupertinoFlingBehaviour(
    coroutineScope: CoroutineScope,
    currentDensity: MutableState<Density>,
) {
    init {
          snapshotFlowOf {currentDensity() }
              .forEach(::updateDensity)
              .collectIn(coroutineScope)
    }
}
```